tests/phpunit/class-wc-stripe-helper-test.php

```
@@ -295,6 +295,74 @@ public function provide_test_get_order_by_intent_id(): array {
 		];
 	}
 
+	/**
+	 * Test for `get_order_by_setup_intent_id`.
+	 *
+	 * @param string|null $intent_target  Which object to save the SetupIntent ID on. One of null, 'refund', or 'order'.
+	 * @param bool        $expect_success Whether an order should be returned.
+	 * @dataProvider provide_test_get_order_by_setup_intent_id
+	 */
+	public function test_get_order_by_setup_intent_id( ?string $intent_target, bool $expect_success ): void {
+		global $wpdb;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$wpdb'. (undefined)

(UnusedLocalVariable)